### PR TITLE
Add helm chart for aws-for-fluent-bit

### PR DESCRIPTION
**Pull request by bnrjee**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/aws/eks-charts/aws-for-fluent-bit/Makefile [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	$(error No gittag was provided.)
+endif
+
+CHART_SCRIPTS_DIR=$(shell git rev-parse --show-toplevel)/helm-charts/scripts
+MAKE_ROOT=$(shell cd "$(dirname "$(BASH_SOURCE[0])")" && pwd -P)
+CLONE_ROOT=$(MAKE_ROOT)/$(AWS_FOR_FLUENT_BIT_CHART_REPO)
+
+.PHONY: clone
+clone: clean
+	git clone  $(AWS_FOR_FLUENT_BIT_CHART_REPO_URL) $(AWS_FOR_FLUENT_BIT_CHART_REPO)
```
Comment (abhay-krishna) [inline projects/aws/eks-charts/aws-for-fluent-bit/Makefile:23]: nit: 1 extra space after `clone`
```suggestion
	git clone $(AWS_FOR_FLUENT_BIT_CHART_REPO_URL) $(AWS_FOR_FLUENT_BIT_CHART_REPO)
```

File: projects/aws/eks-charts/aws-for-fluent-bit/Makefile [hunk 2]
```diff
@@ -0,0 +1,42 @@
+AWS_FOR_FLUENT_BIT_CHART_REPO_URL?=https://github.com/aws/eks-charts/
+AWS_FOR_FLUENT_BIT_CHART_REPO?=eks-charts
```
Comment (abhay-krishna) [inline projects/aws/eks-charts/aws-for-fluent-bit/Makefile:2]: Can you declare the repo name first and then reference the value in the URL like [here](https://github.com/aws/eks-distro-build-tooling/blob/main/projects/prometheus-community/helm-charts/Makefile#L1-L2)? Also can you add `.git` at the end of the clone URI?

**Comment by bnrjee** [comment]
@bnrjee: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
amazon-eks-pod-identity-webhook-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/amazon-eks-pod-identity-webhook-tooling-presubmit/1349417953031884800) | `/test amazon-eks-pod-identity-webhook-tooling-presubmit`

[Full PR test history](https://prow.mhausler.people.a2z.com/pr-history?org=aws&repo=eks-distro-build-tooling&pr=89). [Your PR dashboard](https://prow.mhausler.people.a2z.com/pr?query=is:pr+state:open+author:bnrjee).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by bnrjee** [comment]
@bnrjee: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
amazon-eks-pod-identity-webhook-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/amazon-eks-pod-identity-webhook-tooling-presubmit/1349417953031884800) | `/test amazon-eks-pod-identity-webhook-tooling-presubmit`
release-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/release-tooling-presubmit/1349417953426149376) | `/test release-tooling-presubmit`

[Full PR test history](https://prow.mhausler.people.a2z.com/pr-history?org=aws&repo=eks-distro-build-tooling&pr=89). [Your PR dashboard](https://prow.mhausler.people.a2z.com/pr?query=is:pr+state:open+author:bnrjee).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by bnrjee** [comment]
@bnrjee: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
amazon-eks-pod-identity-webhook-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/amazon-eks-pod-identity-webhook-tooling-presubmit/1349417953031884800) | `/test amazon-eks-pod-identity-webhook-tooling-presubmit`
release-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/release-tooling-presubmit/1349417953426149376) | `/test release-tooling-presubmit`
athens-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/athens-tooling-presubmit/1349417953119965184) | `/test athens-tooling-presubmit`

[Full PR test history](https://prow.mhausler.people.a2z.com/pr-history?org=aws&repo=eks-distro-build-tooling&pr=89). [Your PR dashboard](https://prow.mhausler.people.a2z.com/pr?query=is:pr+state:open+author:bnrjee).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by bnrjee** [comment]
@bnrjee: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
amazon-eks-pod-identity-webhook-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/amazon-eks-pod-identity-webhook-tooling-presubmit/1349417953031884800) | `/test amazon-eks-pod-identity-webhook-tooling-presubmit`
release-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/release-tooling-presubmit/1349417953426149376) | `/test release-tooling-presubmit`
athens-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/athens-tooling-presubmit/1349417953119965184) | `/test athens-tooling-presubmit`
helm-chart-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/helm-chart-tooling-presubmit/1349417953279348736) | `/test helm-chart-tooling-presubmit`

[Full PR test history](https://prow.mhausler.people.a2z.com/pr-history?org=aws&repo=eks-distro-build-tooling&pr=89). [Your PR dashboard](https://prow.mhausler.people.a2z.com/pr?query=is:pr+state:open+author:bnrjee).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by bnrjee** [comment]
@bnrjee: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
amazon-eks-pod-identity-webhook-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/amazon-eks-pod-identity-webhook-tooling-presubmit/1349417953031884800) | `/test amazon-eks-pod-identity-webhook-tooling-presubmit`
release-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/release-tooling-presubmit/1349417953426149376) | `/test release-tooling-presubmit`
athens-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/athens-tooling-presubmit/1349417953119965184) | `/test athens-tooling-presubmit`
helm-chart-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/helm-chart-tooling-presubmit/1349417953279348736) | `/test helm-chart-tooling-presubmit`
aws-for-fluent-bit-helm-chart-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/aws-for-fluent-bit-helm-chart-tooling-presubmit/1349417953182879744) | `/test aws-for-fluent-bit-helm-chart-tooling-presubmit`

[Full PR test history](https://prow.mhausler.people.a2z.com/pr-history?org=aws&repo=eks-distro-build-tooling&pr=89). [Your PR dashboard](https://prow.mhausler.people.a2z.com/pr?query=is:pr+state:open+author:bnrjee).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by bnrjee** [comment]
@bnrjee: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
amazon-eks-pod-identity-webhook-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/amazon-eks-pod-identity-webhook-tooling-presubmit/1349417953031884800) | `/test amazon-eks-pod-identity-webhook-tooling-presubmit`
release-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/release-tooling-presubmit/1349417953426149376) | `/test release-tooling-presubmit`
athens-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/athens-tooling-presubmit/1349417953119965184) | `/test athens-tooling-presubmit`
helm-chart-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/helm-chart-tooling-presubmit/1349417953279348736) | `/test helm-chart-tooling-presubmit`
aws-for-fluent-bit-helm-chart-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/aws-for-fluent-bit-helm-chart-tooling-presubmit/1349417953182879744) | `/test aws-for-fluent-bit-helm-chart-tooling-presubmit`
prometheus-helm-chart-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/prometheus-helm-chart-tooling-presubmit/1349417953375817728) | `/test prometheus-helm-chart-tooling-presubmit`

[Full PR test history](https://prow.mhausler.people.a2z.com/pr-history?org=aws&repo=eks-distro-build-tooling&pr=89). [Your PR dashboard](https://prow.mhausler.people.a2z.com/pr?query=is:pr+state:open+author:bnrjee).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by bnrjee** [comment]
@bnrjee: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
amazon-eks-pod-identity-webhook-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/amazon-eks-pod-identity-webhook-tooling-presubmit/1349417953031884800) | `/test amazon-eks-pod-identity-webhook-tooling-presubmit`
release-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/release-tooling-presubmit/1349417953426149376) | `/test release-tooling-presubmit`
athens-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/athens-tooling-presubmit/1349417953119965184) | `/test athens-tooling-presubmit`
helm-chart-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/helm-chart-tooling-presubmit/1349417953279348736) | `/test helm-chart-tooling-presubmit`
aws-for-fluent-bit-helm-chart-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/aws-for-fluent-bit-helm-chart-tooling-presubmit/1349417953182879744) | `/test aws-for-fluent-bit-helm-chart-tooling-presubmit`
prometheus-helm-chart-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/prometheus-helm-chart-tooling-presubmit/1349417953375817728) | `/test prometheus-helm-chart-tooling-presubmit`
eks-distro-base-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/eks-distro-base-tooling-presubmit/1349417953241600000) | `/test eks-distro-base-tooling-presubmit`

[Full PR test history](https://prow.mhausler.people.a2z.com/pr-history?org=aws&repo=eks-distro-build-tooling&pr=89). [Your PR dashboard](https://prow.mhausler.people.a2z.com/pr?query=is:pr+state:open+author:bnrjee).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by bnrjee** [comment]
@bnrjee: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
amazon-eks-pod-identity-webhook-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/amazon-eks-pod-identity-webhook-tooling-presubmit/1349417953031884800) | `/test amazon-eks-pod-identity-webhook-tooling-presubmit`
release-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/release-tooling-presubmit/1349417953426149376) | `/test release-tooling-presubmit`
athens-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/athens-tooling-presubmit/1349417953119965184) | `/test athens-tooling-presubmit`
helm-chart-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/helm-chart-tooling-presubmit/1349417953279348736) | `/test helm-chart-tooling-presubmit`
aws-for-fluent-bit-helm-chart-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/aws-for-fluent-bit-helm-chart-tooling-presubmit/1349417953182879744) | `/test aws-for-fluent-bit-helm-chart-tooling-presubmit`
prometheus-helm-chart-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/prometheus-helm-chart-tooling-presubmit/1349417953375817728) | `/test prometheus-helm-chart-tooling-presubmit`
eks-distro-base-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/eks-distro-base-tooling-presubmit/1349417953241600000) | `/test eks-distro-base-tooling-presubmit`
prometheus-alertmanager-tooling-presubmit | ea1a611418ec4b3204ab38759665f6c771cf84cc | [link](https://prow.mhausler.people.a2z.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/89/prometheus-alertmanager-tooling-presubmit/1349417953338068992) | `/test prometheus-alertmanager-tooling-presubmit`

[Full PR test history](https://prow.mhausler.people.a2z.com/pr-history?org=aws&repo=eks-distro-build-tooling&pr=89). [Your PR dashboard](https://prow.mhausler.people.a2z.com/pr?query=is:pr+state:open+author:bnrjee).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by bnrjee** [comment]
test /aws-for-fluent-bit-helm-chart-tooling-presubmit

**Comment by abhay-krishna** [comment]
test /aws-for-fluent-bit-helm-chart-tooling-presubmit

File: projects/aws/eks-charts/aws-for-fluent-bit/Makefile [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+AWS_FOR_FLUENT_BIT_CHART_REPO_URL?=https://github.com/aws/$(AWS_FOR_FLUENT_BIT_CHART_REPO).git
+AWS_FOR_FLUENT_BIT_CHART_GIT_TAG?=v0.0.41
+
+ifeq ("$(AWS_FOR_FLUENT_BIT_CHART_REPO_URL)","")
+        $(error No repository URL was provided.)
```
Comment (vivek-koppuru) [inline projects/aws/eks-charts/aws-for-fluent-bit/Makefile:6]: Is this indented different than the others?

Comment (bnrjee) [reply projects/aws/eks-charts/aws-for-fluent-bit/Makefile:6]: Fixed

**Review by vivek-koppuru — APPROVED** [review]
/lgtm

**Pull request merged**